### DISCUSSION TITLE
How to get the old "Type here to search" box instead of the search icon

### DISCUSSION THREAD

**Discussion by junaayd123** [valinet/ExplorerPatcher#2552] [Q&A]
the search box icon is just the magnifying glass, however I want it to be the actual "type here to search"

**Comment by Gargantubrain** [comment]
This appears to be a duplicate of [#2545 ](https://github.com/valinet/ExplorerPatcher/discussions/2545)

**Reply by pyrates999** [reply]
They also stopped responding when asked for details.

**Comment by milestprower92** [comment]
> the search box icon is just the magnifying glass, however I want it to be the actual "type here to search"

No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.
